packages/perf/lib/ScreenTrace.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2016-present Invertase Limited & Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this library except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+let id = 0;
+
+export default class ScreenTrace {
+  constructor(native, identifier) {
+    this.native = native;
+    this._identifier = identifier;
+    this._id = id++;
+    this._started = false;
+    this._stopped = false;
+  }
+
+  start() {
+    if (this._started) {
+      return Promise.resolve(null);
+    }
+    this._started = true;
+
+    return this.native.startScreenTrace(this._id, this._identifier);
+  }
+
+  stop() {
+    if (this._stopped) {
```

Comment:
maybe 
```suggestion
    if (!this._started || this._stopped) {
```

...that way it is safe to call stop() if start never worked, this makes stop safe even if on iOS where start won't work, or on unsupported platforms where start doesn't work

Reply:
fixed